app/utils/discovery.ts

```
@@ -0,0 +1,127 @@
+
+    if (!response.ok) {
+      throw new Error(
+        `Failed to fetch Plex library sections: ${response.status} ${response.statusText}`
+      );
+    }
+
+    const data = await response.json();
+
+    // Extract sections from the response
+    const sectionsData =
+      data.MediaContainer?.Directory?.map((section: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)